invoices/views.py

```diff
@@ -279,6 +308,8 @@ def queue_update(request):
 def get_pdf(request, invoice_id, pdf_type):
     if pdf_type == "hours":
         pdf, title = generate_hours_pdf_for_invoice(request, invoice_id)
+    elif pdf_type == "weekly":
+        pdf, title = generate_hours_pdf_for_weekly_report(request, invoice_id) #invoice_id is actually weekly_report_id
```

Comment:
> invoice_id is actually weekly_report_id

This is not a proper way to do this - if the argument name says it is `invoice_id`, it won't be something else.